config/strings.yaml

```diff
@@ -80,3 +80,8 @@ error:
     heading: Uh oh, that's a 500.
     message: Perhaps try to search for something else instead?
 
+# warning messages
+warning:
+  duplicateDoc: "Warning: Multiple documents in this folder share the same name as this document. Only one of the documents will be accesible through Library."
```

Comment:
It looks like we don't know for sure whether the other item in the drive that has the exact same slug as this item is the same type; you could mix and match.

A little more complex than a static warning message, but what if we made the warning include the names of the other items that would have shown up at this location to make it easier to resolve the issue? We could save the names of the other documents (or folders!) where we currently set  `isDuplicate`, then pass them into a yaml function like in the `moveFile` definition. In that case, we might want to discard the boolean and instead create an array called `duplicates` as a property.

The general logic of the diff wouldn't change much, but we could drop the ternary in `nav.ejs` and be more specific without needing as many configuration options in strings.yaml.

Reply:
Good idea! I'll make the changes.